changelog.md

```diff
@@ -1,2 +1,11 @@
-#v0.1
+v0.1.1
+------
+
+* Fix a deployment bug by adding `MANIFEST.in`.
+* Add a missing requirement for `django`.
```

Comment:
(And yeah, ignore the `0.1.1` stuff, that's just a changelog update for release.)